### ISSUE TITLE
It's about video speed on YouTube

### ISSUE THREAD

**Issue by Jottarro** [brave/brave-browser#35652]
Let's log in to youtube from Brave, open any video and change the speed of the video, for example 1.5x, then press and hold the screen and make the speed of the video 2x, when we stop holding down, the speed of the video returns to normal speed, which should be the speed I chose last time, ie 1.5x

**Comment by codey-badger** [comment]
Yeah it's very annoying, did you find any workaround in the mean time?
